routes/v1.js

```diff
@@ -33,12 +34,32 @@ apiRoutes.route('/verify/:verificationId')
   .get(UserController.verifyAccount);
 
 apiRoutes.route('/articles')
-  .post(Auth.verifyUser, isUserVerified,
+  .post(
+    Auth.verifyUser, isUserVerified,
     addImages,
     validateArticle,
     returnValidationErrors,
     generateSlug,
-    createArticle);
+    createArticle
+  );
+
+apiRoutes.route('/articles/:slug')
+  .put(
+    Auth.verifyUser, isUserVerified,
+    validateUser,
+    addImages,
+    validateArticle,
+    returnValidationErrors,
+    updateArticle,
+    generateSlug
```

Comment:
@Dubby20 why do we need to generate a slug here again?

Reply:
@9jaswag the documentation says we need to generate `slug` when we update the title of an article.
<img width="480" alt="Screenshot 2019-03-18 at 1 59 50 PM" src="https://user-images.githubusercontent.com/29398098/54531808-793d0c00-4986-11e9-984d-94f80739475b.png">